tests/integration/tests/test_dualstack.py

```diff
@@ -16,22 +17,24 @@ def test_dualstack(h: harness.Harness, tmp_path: Path):
     snap_path = (tmp_path / "k8s.snap").as_posix()
     main = h.new_instance(dualstack=True)
     util.setup_k8s_snap(main, snap_path)
-    bootstrap_dualstack_config_path = "/home/ubuntu/bootstrap-dualstack.yaml"
-    main.send_file(
-        (config.MANIFESTS_DIR / "bootstrap-dualstack.yaml").as_posix(),
-        bootstrap_dualstack_config_path,
-    )
 
-    main.exec(["k8s", "bootstrap", "--file", bootstrap_dualstack_config_path])
+    with open(config.MANIFESTS_DIR / "bootstrap-dualstack.yaml", "r") as file:
+        bootstrap_config = yaml.safe_load(file)
```

Comment:
btw, this can be `yaml.safe_load(config.MANIFESTS_DIR / "bootstrap-dualstack.yaml").read_text())`, same below